lib/index.js

```diff
@@ -15,16 +16,31 @@ export const metadata = {
   version: require('../package.json').version
 };
 
+export const mapComponentDispatch = {
+  Header: (dispatch, map) =>
+    Object.assign(map, {
+      setCurrentClient: info => dispatch(setCurrentClient(info))
+    })
+};
+
 export const mapComponentState = {
   Header: (state, map) =>
     Object.assign(map, {
-      bcoinUri: state.node.serverInfo.bcoinUri,
```

Comment:
Are you sure that this isn't being used anywhere?

Reply:
Not in this plugin which is all that this affects.